Source/Applications/Backend/listing/ApplicationForListing.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Field, ObjectType } from 'type-graphql';
+import { guid } from '@shared/backend/data';
+import { prop, getModelForClass } from '@typegoose/typegoose';
+import { Guid } from '@dolittle/rudiments';
+import { MicroserviceForListing } from './MicroserviceForListing';
```

Comment:
We don't need all of these:
```suggestion
import { Field, ObjectType } from 'type-graphql';
import { prop, getModelForClass } from '@typegoose/typegoose';
import { MicroserviceForListing } from './MicroserviceForListing';
```